modules/ROOT/pages/salesforce/salesforce-composite-connector.adoc

```diff
@@ -9,7 +9,7 @@ Salesforce Composite Connector v2.3
 
 Anypoint Connector for Salesforce Composite enables you to work with the Salesforce Batch and SObject Tree APIs. The connector exposes convenient methods for preparing subordinate requests, executing them in one batch, and parsing the results. Also it offers the posibility of creating SObject Trees through an operation separate from the batch ones, called createSobjectTree. This connector provides DataSense for preparing subordinate requests and parsing responses. DataSense is also provided for execution, and for creating SObject trees.
 
-Release Notes: xref:release-notes::connector/salesforce-composite-connector-release-notes-mule-4.adoc[Salesforce Composite Connector Release Notes] + 
+Release Notes: xref:release-notes::connector/salesforce-composite-connector-release-notes-mule-4.adoc[Salesforce Composite Connector Release Notes] +
```

Comment:
Should we update to full name, or is that done in the narrative directly following the title?